lib/utils/helpers.ts

```diff
@@ -115,7 +114,7 @@ export function runCommand<T>(command: string): Promise<T> {
 		require('../app-capitano');
 
 		const capitanoRunAsync = promisify(capitano.run);
-		return capitanoRunAsync(command);
+		return capitanoRunAsync(commandArgs.join(' '));
```

Comment:
I was assuming we would be able to avoid `join(' ')` as well. I think we can:
* https://github.com/balena-io/capitano/blob/v1.9.2/lib/capitano.coffee#L59-L60
* https://github.com/balena-io/capitano/blob/v1.9.2/lib/parse.coffee#L11

Hopefully it is sufficient to edit the following typings file in the `balena-cli` repo:
* https://github.com/balena-io/balena-cli/blob/v12.3.12/typings/capitano/index.d.ts#L85

`command: string` → `command: string | string[]`

Then:
```
		return capitanoRunAsync(commandArgs);
```

Reply:
Ah nice, will try that.

Reply:
Yep, appears to work.